client/src/chess/auth/auth.component.js

```diff
@@ -1,6 +1,12 @@
 module.exports = {
-  controller: AuthController,
-  templateUrl: 'auth/auth.component.html'
+  login : {
```

Comment:
do not export two components from one file: slit it to two files. if you need the same controller (this is very strange) - just move it no separate file